source/includes/_errors.md

```diff
@@ -1,36 +1,68 @@
 ## Errors
+Retrieves information for errors and discrepancy in the Metislabs API. This should provide the user with information on why certain `Readings` or `Predictions` are missing.
```

Comment:
discrepancies?

Reply:
Also maybe not describe it as being errors in the Metislabs API, as often they're external errors (e.g. their historian isn't providing data)